airflow/contrib/operators/gcp_translate_operator.py

```diff
@@ -0,0 +1,119 @@
+    This is a list of dictionaries for each queried value. Each
+    dictionary typically contains three keys (though not
+    all will be present in all cases)
+
+    * ``detectedSourceLanguage``: The detected language (as an
+      ISO 639-1 language code) of the text.
+    * ``translatedText``: The translation of the text into the
+      target language.
+    * ``input``: The corresponding input value.
+    * ``model``: The model used to translate the text.
+
+    If only a single value is passed, then only a single
+    dictionary is set as XCom return value
+
+    :type values: str or list
+    :param values: String or list of strings to translate.
+
+    :type target_language: str
+    :param target_language: The language to translate results into. This
+      is required by the API and defaults to
+      the target language of the current instance.
+
+    :type format_: str or None
+    :param format_: (Optional) One of ``text`` or ``html``, to specify
+      if the input text is plain text or HTML.
+
+    :type source_language: str or None
+    :param source_language: (Optional) The language of the text to
+      be translated.
+
+    :type customization_ids: str or list or None
+    :param customization_ids: (Optional) ID or list of customization IDs
+      for translation. Sets the ``cid`` parameter
+      in the query.
+
+    :type model: str or None
+    :param model: (Optional) The model used to translate the text, such
+      as ``'base'`` or ``'nmt'``.
+
+    """
+
+    # [START translate_template_fields]
+    template_fields = ('target_language', 'format_', 'source_language', 'model', 'gcp_conn_id')
+    # [END translate_template_fields]
+
+    @apply_defaults
+    def __init__(
+        self,
+        values,
+        target_language,
+        format_,
+        source_language,
+        customization_ids,
+        model,
+        gcp_conn_id='google_cloud_default',
+        *args,
+        **kwargs
+    ):
+        super(CloudTranslateTextOperator, self).__init__(*args, **kwargs)
+        self.values = values
+        self.target_language = target_language
+        self.format_ = format_
+        self.source_language = source_language
+        self.customization_ids = customization_ids
+        self.model = model
+        self.gcp_conn_id = gcp_conn_id
+
+    def execute(self, context):
+        _hook = CloudTranslateHook(gcp_conn_id=self.gcp_conn_id)
+        try:
+            translation = _hook.translate(
+                values=self.values,
+                target_language=self.target_language,
+                format_=self.format_,
+                source_language=self.source_language,
+                customization_ids=self.customization_ids,
+                model=self.model,
+            )
+            self.log.debug("Translation %s", translation)
+            return translation
+        except ValueError as e:
+            self.log.error('An error has been thrown from translate method:', e)
```

Comment:
```suggestion
            self.log.error('An error has been thrown from translate method:\n', e)
```
Maybe we could print the error details in a new line?

Reply:
Why not. fixed :)